src/Config.php

```diff
@@ -241,4 +231,13 @@ public function setAlias(string $name, string $class) : self
         $this->types[$name]['typeOf'] = $class;
         return $this;
     }
+
+    private function ensureArrayOrArrayAccess($options) : void
+    {
+        if (! is_array($options) && ! ($options instanceof ArrayAccess)) {
```

Comment:
Can be simplified via de Morgan's Law:

`if (! (is_array($options) || $options instanceof ArrayAccess)) {`

Also, the `instanceof` check is a bit lighter